shaders/clouds.omwfx

```diff
@@ -637,6 +614,33 @@ fragment main {
     }
 
     void main() {
+
+        //Array of cloud densities per weather ID
+        float cloud_transition[10];
+        cloud_transition[0] = float(0.2);  //Clear
+        cloud_transition[1] = float(0.35); //Cloudy
+        cloud_transition[2] = float(0.25);  //Foggy
+        cloud_transition[3] = float(0.4);  //Overcast
+        cloud_transition[4] = float(0.5);  //Rain
+        cloud_transition[5] = float(0.6);  //Thunder
+        cloud_transition[6] = float(0.7);  //Ash
+        cloud_transition[7] = float(0.7);  //Blight
+        cloud_transition[8] = float(0.5);  //Snow
+        cloud_transition[9] = float(0.6);   //Blizzard
+
+        //Array of mist densities per weather ID
+        float mist_transition[10];
+        mist_transition[0] = float(0.1);  //Clear
+        mist_transition[1] = float(0.25);  //Cloudy
+        mist_transition[2] = float(0.75);  //Foggy
+        mist_transition[3] = float(0.35);  //Overcast
+        mist_transition[4] = float(0.4);  //Rain
+        mist_transition[5] = float(0.4);  //Thunder
+        mist_transition[6] = float(0.3);  //Ash
+        mist_transition[7] = float(0.2);  //Blight
+        mist_transition[8] = float(0.3);  //Snow
+        mist_transition[9] = float(0.4);   //Blizzard
+
```

Comment:
Why not put these into dedicated functions too like `weather_tone`? At least then the degree to which the compiler can optimise things is consistent for each GL impl.

Reply:
Ok, changed them to a function.